packages/noports_core/lib/src/sshnp/models/sshnp_params.dart

```diff
@@ -319,8 +379,12 @@ class SshnpPartialParams {
   factory SshnpPartialParams.fromArgMap(Map<String, dynamic> args) {
     return SshnpPartialParams(
       profileName: args[SshnpArg.profileNameArg.name],
-      clientAtSign: args[SshnpArg.fromArg.name],
-      sshnpdAtSign: args[SshnpArg.toArg.name],
+      clientAtSign: args[SshnpArg.fromArg.name] == null
+          ? null
+          : AtUtils.fixAtSign(args[SshnpArg.fromArg.name]),
+      sshnpdAtSign: args[SshnpArg.toArg.name] == null
+          ? null
+          : AtUtils.fixAtSign(args[SshnpArg.toArg.name]),
```

Comment:
Need atSigns to consistently have the `@` prefixed

Reply:
We dropped support for non-atSign hosts, so I think we can enforce the check for those too.